utils/backup/S3/fo-backup-s3

```diff
@@ -0,0 +1,43 @@
+#!/bin/bash
+# Copyright (C) Siemens AG 2017
+# This program is free software; you can redistribute it and/or
+# modify it under the terms of the GNU General Public License
+# version 2 as published by the Free Software Foundation.
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+# You should have received a copy of the GNU General Public License along
+# with this program; if not, write to the Free Software Foundation, Inc.,
+# 51 Franklin Street, Fifth Floor, Boston, MA  02110-1301, USA.
+
+DIR="$( cd "$( dirname "${BASH_SOURCE[0]}" )" && pwd )"
+source ${DIR}/before
+set -x
+${DIR}/install
+
+echo Starting backup at `date`
+
+
+
+echo Creating diff of repository at `date`
+ls -RlahGg --time-style='+' /srv/fossology/repository/ | cut -d ' ' -f 2-5 | ${DIR}/venv/bin/aws s3 cp --sse AES256 - "s3://${BUCKET}/${FILENAME}-repo-diff.txt"
```

Comment:
# example:
```
$  tree ./                                                                                                                                                                                                                               /tmp/test
./
├── Folder with splaces
│   └── file
└── otherFile
```
and
```
$ ls -RlahGg --time-style='+' . | cut -d ' ' -f 2-5                                                                                                                                                                                     /tmp/test
.:
20K
 3 4,0K 
16  12K 
 2 4,0K 
 1  

with splaces:
8,0K
2 4,0K  .
3 4,0K  ..
1  
```

## Note:
here you also see that the `ls -RlahGg --time-style='+'` contains on my system spaces and not tabs => the filenames are not listed.